packages/vit/package.json

```diff
@@ -45,5 +45,9 @@
     "@types/mustache": "^4.1.1",
     "react": "^17.0.2",
     "vite": "^2.1.3"
+  },
+  "peerDependencies": {
+    "globby": "^11.x",
+    "react": "^17.x"
```

Comment:
Due to `globby` dep had fixed, `globby` can remove from `peerDependencies`. `react: 16.x || 17.x` should be better?

Reply:
Yeah, so much better )